data/hai/hai reveal 1 intro.txt

```diff
@@ -542,7 +540,7 @@ mission "Hai Leaks Response 1"
 				`	"That seems practical."`
 				`	"I don't really have an option to refuse, do I?"`
 			`	"Fantastic, you understand the situation," she says, though her tone is much more businesslike than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments, then we can begin a dialogue on how best to deal with this current scenario."`
-			`	She hands you an officially branded box containing what appears to be a diplomatic ID. The text on the ID appears to identify you in a few scripts you've seen in human space and definitely recognize as human, but it also has another script you've never seen before that must be in Hai. You wonder for a brief moment how they were able to gain this information on you ahead of time.` # Perhaps the ID could be a license that allows unrestricted access to Hai-home. If anything it just makes this bit meaningful. 
+			`	She hands you an officially branded box containing what appears to be a diplomatic ID. The text on the ID appears to identify you in a few scripts you've seen in human space and definitely recognize as human, but it also has another script you've never seen before that must be in Hai. You wonder for a brief moment how they were able to gain this information on you ahead of time.` 
```

Comment:
It has occurred to me in reviewing certain things through that this ID wouldn't have "script" on it. It would either be some kind of badge/seal of a specific unforgeable design that simply denotes you as an emissary of the Hai, or it would be digital of some kind - potentially holographic. Probably keyed to recognise spoken language and display its information in that language (if it's one that it recognises).